boards/atmel/sam/sam4e_xpro/sam4e_xpro.dts

```diff
@@ -52,75 +52,75 @@
 		compatible = "atmel-xplained-pro-header";
 		#gpio-cells = <2>;
 		gpio-map-mask = <0xffffffff 0xffffffc0>;
-		gpio-map-pass-thru = <0 0x3f>;		/*           Shared */
-		gpio-map =	<0  0 &piob  2 0>,	/* AFE AD0          */
-				<1  0 &piob  3 0>,	/* AFE AD1          */
-				<2  0 &pioa 24 0>,	/* GPIO             */
-				<3  0 &pioa 25 0>,	/* GPIO             */
-				<4  0 &pioa 15 0>,	/* TIOA1            */
-				<5  0 &pioa 16 0>,	/* TIOB1            */
-				<6  0 &pioa 11 0>,	/* WKUP7            */
-				<7  0 &piod 25 0>,	/* GPIO             */
-				<8  0 &pioa  3 0>,	/* TWD0        EXTx */
-				<9  0 &pioa  4 0>,	/* TWCK0       EXTx */
-				<10 0 &pioa 21 0>,	/* RXD1             */
-				<11 0 &pioa 22 0>,	/* TXD1             */
-				<12 0 &piob 14 0>,	/* SPI(NPCS1)       */
-				<13 0 &pioa 13 0>,	/* SPI(MOSI)   EXTx */
-				<14 0 &pioa 12 0>,	/* SPI(MISO)   EXTx */
-				<15 0 &pioa 14 0>;	/* SPI(SCK)    EXTx */
-							/* GND              */
-							/* +3.3V            */
+		gpio-map-pass-thru = <0 0x3f>; /*           Shared */
+		gpio-map = <0 0 &piob 2 0>, /* AFE AD0          */
```

Comment:
Comments should stay vertically align for all boards.

Reply:
This was already discussed and a space is to be used after a comments. I users want to keep alignment then // dts-format off // dts-format on is to be dded around the block

Reply:
I really don't like when people give more rights to tools then people.
Fix the tool or add the dts-format off/on because comment is for people and machines should ignore that.